package.json

```diff
@@ -10,10 +10,10 @@
     "ts:transpile": "tsc --project tsconfig.build.json",
     "clean-dist": "rimraf \"packages/**/dist\" && rimraf \".bob\"",
     "build": "yarn ts:transpile && bob build",
-    "build:api-docs": "node scripts/build-api-docs.js",
+    "build:api-docs": "ts-node --compiler-options='{\"module\":\"commonjs\"}' scripts/build-api-docs",
     "lint": "eslint --ext .ts .",
-    "prettier": "prettier --ignore-path .gitignore --ignore-path .prettierignore --write --list-different .",
```

Comment:
`--ignore-path` accept only the last flag, so calling `--ignore-path .gitignore` make nothing